doc/linux.t2t

```diff
@@ -133,10 +133,23 @@ Now on with the build:
 
 ```
 make
```

Comment:
From my point of view it is ``make -j 8`` :P 

Note that you can also use ``make -j 8 qgis``, compile only things needed by qgis executable, this can be useful for development, when no need to compile everything.